src/shared/components/TextArea/TextArea.style.ts

```diff
@@ -0,0 +1,61 @@
+import { colors, sizes, transitions, typography } from '@/shared/theme'
+import styled from '@emotion/styled'
+
+export const TextAreaWrapper = styled.div`
+  max-width: 300px;
+`
+
+export const StyledTextArea = styled.textarea`
+  background: ${colors.black};
+  width: 100%;
+  resize: none;
+  color: ${colors.gray[300]};
+  font-size: ${typography.sizes.subtitle2};
+  border: none;
+  border-bottom: 1px solid ${colors.gray[900]};
+  transition: all ${transitions.timings.loading} ${transitions.easing};
+  ::placeholder {
+    color: ${colors.gray[400]};
+  }
+  &:hover {
+    background: ${colors.gray[900]};
```

Comment:
I think we will need to keep it at the `transparent` background at all times. Otherwise, we'd need some padding within the text area. With the colored background you can clearly see the area's boundary which looks visually unsexy without padding